src/streamlink/plugins/twitch.py

```diff
@@ -212,58 +200,84 @@ def _gql_persisted_query(cls, operationname, sha256hash, **variables):
             "variables": dict(**variables)
         }
 
-    # Public API calls
-
-    def channel_from_video_id(self, video_id):
-        return self.call("/kraken/videos/{0}".format(video_id), schema=validate.Schema(
-            {"channel": {
-                "_id": validate.transform(int),
-                "name": validate.all(str, validate.transform(lambda n: n.lower()))
-            }},
-            validate.get("channel"),
-            validate.union_get("_id", "name")
+    @staticmethod
+    def parse_token(tokenstr):
+        return parse_json(tokenstr, schema=validate.Schema(
+            {"chansub": {"restricted_bitrates": validate.all(
+                [str],
+                validate.filter(lambda n: not re.match(r"(.+_)?archives|live|chunked", n))
+            )}},
+            validate.get(("chansub", "restricted_bitrates"))
         ))
 
-    def channel_from_login(self, channel):
-        return self.call("/kraken/users", login=channel, schema=validate.Schema(
-            {"users": [{
-                "_id": validate.transform(int)
-            }]},
-            validate.get(("users", 0, "_id"))
-        ))
+    # GraphQL API calls
 
     def metadata_video(self, video_id):
-        return self.call("/kraken/videos/{0}".format(video_id), schema=validate.Schema(validate.any(
-            validate.all(
-                {
-                    "title": str,
-                    "game": str,
-                    "channel": {"display_name": str}
+        query = self._gql_persisted_query(
+            "VideoMetadata",
+            "cb3b1eb2f2d2b2f65b8389ba446ec521d76c3aa44f5424a1b1d235fe21eb4806",
+            channelLogin="",  # parameter can be empty
+            videoID=video_id
+        )
+
+        return self.call(query, schema=validate.Schema(
+            {"data": {"video": {
+                "owner": {
+                    "displayName": str
                 },
-                validate.union_get(("channel", "display_name"), "title", "game")
+                "title": str,
+                "game": {
+                    "displayName": str
+                }
+            }}},
+            validate.get(("data", "video")),
+            validate.union_get(
+                ("owner", "displayName"),
+                "title",
+                ("game", "displayName")
+            )
+        ))
+
+    def metadata_channel(self, channel):
+        queries = [
+            self._gql_persisted_query(
+                "ChannelShell",
+                "c3ea5a669ec074a58df5c11ce3c27093fa38534c94286dc14b68a25d5adcbf55",
+                login=channel,
+                lcpVideosEnabled=False
```

Comment:
I couldn't find any other simple GQL queries for extracting the channel's displayName that don't have any weird request parameters/variables. `ChannelShell` is not ideal, but it seemed to be the best fit.